base/error.go

```diff
@@ -67,6 +67,9 @@ var (
 
 	// ErrConfigRegistryReloadRequired is returned when a db config fetch requires a registry reload based on version mismatch (config is newer)
 	ErrConfigRegistryReloadRequired = &sgError{"Config registry reload required"}
+
+	// ErrMaximumChannelsForUserExceeded is returned when running in serverless mode and the user has more than 500 channels granted to them
+	ErrMaximumChannelsForUserExceeded = &sgError{"User has exceeded maximum of 500 channels"}
```

Comment:
Let's move `ServerlessChannelLimit` to base and then we can use this constant here, otherwise it runs the risk of being out of date